docs/release-notes/0.11.0.md

```diff
@@ -5,9 +5,9 @@
 
 * Allow `axis` parameter of e.g. :func:`anndata.concat` to accept `'obs'` and `'var` {pr}`1244` {user}`flying-sheep`
 * Add `settings` object with methods for altering internally-used options, like checking for uniqueness on `obs`' index {pr}`1270` {user}`ilan-gold`
-* Add `should_remove_unused_categories` option to `anndata.settings` to override current behavior.  Default is `True` (i.e., previous behavior).  Please refer to the [documentation](https://anndata.readthedocs.io/en/latest/generated/anndata.settings.html) for usage.  {pr}`1340` {user}`ilan-gold`
 * `scipy.sparse.csr_array` and `scipy.sparse.csc_array` are now supported when constructing `AnnData` objects {pr}`1028` {user}`ilan-gold` {user}`isaac-virshup`
-* Add `should_check_uniqueness` option to `anndata.settings` to override current behavior.  Default is `True` (i.e., previous behavior).  Please refer to the [documentation](https://anndata.readthedocs.io/en/latest/generated/anndata.settings.html) for usage.  {pr}`1507` {user}`ilan-gold`
+* Add {attr}`~anndata.settings.should_remove_unused_categories` option to `anndata.settings` to override current behavior {pr}`1340` {user}`ilan-gold`
+* Add {attr}`~anndata.settings.should_check_uniqueness` option to `anndata.settings` to override current behavior {pr}`1507` {user}`ilan-gold`
```

Comment:
link internal objects like this.

instead of `[...](https://anndata.readthedocs.io/en/latest/generated/anndata.settings.html)`, you should do ``{mod}`anndata.settings` ``